test/e2e/node/node_problem_detector.go

```diff
@@ -41,10 +41,11 @@ import (
 
 // This test checks if node-problem-detector (NPD) runs fine without error on
 // the nodes in the cluster. NPD's functionality is tested in e2e_node tests.
-var _ = SIGDescribe("NodeProblemDetector [DisabledForLargeClusters]", func() {
+var _ = SIGDescribe("NodeProblemDetector", func() {
 	const (
-		pollInterval = 1 * time.Second
-		pollTimeout  = 1 * time.Minute
+		pollInterval      = 1 * time.Second
+		pollTimeout       = 1 * time.Minute
+		maxNodesToProcess = 10
```

Comment:
Do we want to fix the comment in lines 42/43 to say that we will check up-to-max-of-10-nodes

Reply:
done